projects/storefrontlib/cms-components/navigation/navigation/navigation-node.model.ts

```diff
@@ -31,3 +33,16 @@ export interface NavigationNode {
    */
   styleAttributes?: string;
 }
+
+export const ICONS: Record<string, string> = {
+  'Orders And Returns': ICON_TYPE.CART,
+  'Saved Carts': ICON_TYPE.CART_PLUS,
+  Wishlists: ICON_TYPE.STAR,
+  Requests: ICON_TYPE.HEADSET,
+  'Personal Details': ICON_TYPE.USER,
+  'Password And Security': ICON_TYPE.PASSWORD,
+  'Address Book': ICON_TYPE.HOME,
+  'Payment Details': ICON_TYPE.CREDIT_CARD,
+  Communications: ICON_TYPE.COMMUNICATIONS,
+  'Privacy And Settings': ICON_TYPE.PRIVACY,
+};
```

Comment:
[MEDIUM]
This is a set of icons needed only for my account feature. Why then it's polluting a generic `navigation-node.model` file in the core library of `@spartacus/storefront`?
Moreover, it looks strange to have those my-account-specific icons exported in the public API of `@spartacus/storefront` as a generic item name `ICONS` (.e.g. in such case customers would do `import { ICONS } from '@spartacus/storefront'`. 
What's more, we an enum [ICON_TYPE](https://github.com/SAP/spartacus/blob/14a1af8862eb22363303e2f30982034bef4f3639/projects/storefrontlib/cms-components/misc/icon/icon.model.ts#L11) exported from this library. Which can be confusing for customers and ourselves. (Note: I'd like to avoid putting those my-account-specific keys into the generic `ICON_TYPE` enum).

Reply:
hi @Platonn , can i rename ICONS to MYACCOUNT_ICONS or do you mean creating a new fontawesomeIconConfig/enum 
 for my-account